gpytorch/utils/linear_cg.py

```diff
@@ -55,6 +97,9 @@ def linear_cg(
         initial_guess = torch.zeros_like(rhs)
     if preconditioner is None:
         preconditioner = _default_preconditioner
+        no_precond = True
```

Comment:
Nit: This seems a little odd with the double negative below. How about just using precond as the flag?